src/clr-angular/layout/tabs/tab-content.ts

```diff
@@ -12,16 +12,17 @@ let nbTabContentComponents: number = 0;
 @Component({
   selector: 'clr-tab-content',
   template: `
+    <ng-template #tabContentProjectedRef>
+      <section [id]="tabContentId" role="tabpanel" [class.active]="active"
+               [hidden]="!active"
+               [attr.aria-labelledby]="ariaLabelledBy"
+               [attr.aria-expanded]="active"
+               [attr.aria-hidden]="!active" 
+               [attr.data-hidden]="!active">
```

Comment:
Do we need `data-hidden`? Thats just something you see more with Bootstrap and isn't necessary I believe.

Reply:
I restored it from git history like this. Even if it had been needed some time ago, there is no obvious reason to keep it now. Removing it.